packages/nameguard-react/src/components/ReportIcon/index.tsx

```diff
@@ -8,11 +8,11 @@ import {
 import cc from "classcat";
 
 import { Tooltip } from "../Tooltip/Tooltip";
-import { RatingLoadingIcon } from "../icons/RatingLoadingIcon";
 import { RatingIcon, RatingIconSize } from "../Report/RatingIcon";
 import { checkResultCodeTextColor, ratingTextColor } from "../../utils/text";
 import { ENSName } from "@namehash/ens-utils";
 import { UnknownReportIcon } from "../UnknownReportIcon/UnknownReportIcon";
```

Comment:
Please see my other comments about naming standards. It seems maybe we should also rename `UnknownReportIcon` to `ReportUnknownIcon`?